source/index.html.md

```diff
@@ -27,6 +27,10 @@ Please note that as we are still finalizing our API, this spec is subject to cha
 
 # Change Log
 
+## 7 December 2021
+
+Added `customer_hashed_email_address` to the order payload.
```

Comment:
worth mentioning explicitly that it's contained within a metadata object here?

Reply:
true. lemmy do that